.eslintrc.js

```diff
@@ -105,6 +105,7 @@ module.exports = {
         ],
       },
     ],
+    'import/no-unresolved': [2, {ignore: ['fsevents']}],
```

Comment:
Yea, this fixes the issue. However, maybe it would be better to mark it as dev dependency, and remove from `optionalDependencies`? Same as we use `weak-napi`. cc @SimenB

Reply:
I think it should not be in `devDependencies` because it might throw an error during installation on non-macOS systems.